test/ring/middleware/webjars_test.clj

```diff
@@ -19,21 +19,26 @@
     (handler request respond raise)
     @respond))
 
+(defn- slurp-async-response [handler uri]
```

Comment:
Why have these changes been added to the test namespace?

Reply:
I was trying to follow the boy scout rule (some DRYing up), and also thought it would be easier to grok the cases as examples if each group's test cases were uniform. I can revert these and just add the new test cases matching the previous style if you prefer it that way.